tequilapi/endpoints/service.go

```diff
@@ -54,10 +54,22 @@ type serviceInfo struct {
 	// example: 6ba7b810-9dad-11d1-80b4-00c04fd430c8
 	ID string `json:"id"`
 
-	Proposal proposalRes `json:"proposal"`
+	// provider identity
+	// example: 0x0000000000000000000000000000000000000002
+	ProviderID string `json:"providerId"`
+
+	// service type. Possible values are "openvpn", "wireguard" and "noop"
+	// example: openvpn
+	Type string `json:"type"`
+
+	// options with which service was started. Every service has a unique list of allowed options.
```

Comment:
we could expand on this comment, adding a link to where you could find the unique list - if we had one.

Reply:
Not my scope here. We're thinking to implement emdpoint `/service/types` which will expose service types and details how to start them